doc/pages/expansions.asciidoc

```diff
@@ -218,6 +218,23 @@ The following expansions are supported (with required context _in italics_):
     _in window scope_ +
     line of the main cursor, the third component of `%val{selection_desc}`
 
+*%val{history}*::
+    _in buffer, window scope_ +
+    the full change history of the buffer, including undo forks, as a
+    _history_id_ followed by one or more history entries. _history_id_ may
+    be different from `%val{history_id}` to capture uncommitted changes, for
+    example from an InsertChar hook. Each entry is
+    `parent committed redo_child modification0 modification1 ...`.
+    _parent_ is the index of the entry's predecessor.  Entry 0, which is the
+    root of the history tree, will always have `-` for _parent_. _committed_
+    is the count of seconds from January 1, 1970 to the creation of the
+    history entry, or `-` if the entry represents uncommitted changes.
+    _redo_child_ is the index of the child which will be visited for `U` and
+    will be `-` at the leaves of the history. Each _modification_ is in the
+    format `op|line.column|text`, where _op_ is `+` for insertions or `-`
+    for deletions, _line_ and _column_ are the 1-based position of the start
+    of the change, and _text_ is the content of the insertion or deletion.
```

Comment:
"column" gets messy when you have files with code-points that can take two or zero columns on-screen. It would be better to define these as "in the same format used for the anchor and cursor positions in `%val{selection_desc}`" or "in the same format used for the anchor and cursor positions in the `:select` command" or similar.

Reply:
done.